fastai/vision/learner.py

```diff
@@ -112,6 +112,43 @@ def plot_top_losses(self, k, largest=True, figsize=(12,12)):
             cl = int(cl)
             im.show(ax=axes.flat[i], title=
                 f'{classes[self.pred_class[idx]]}/{classes[cl]} / {self.losses[idx]:.2f} / {self.probs[idx][cl]:.2f}')
+            
+    def plot_multi_top_losses(self, samples:int=3, figsz:Tuple[int,int]=(8,8)):
+        "Show images in `top_losses` along with their prediction, actual, loss, and probability of predicted class in a multilabeled dataset."
+        if samples >20:
+            print("Max 20 samples")
+            return
+        losses, idxs = self.top_losses(self.data.c)
+        infolist, ordlosses_idxs, mismatches_idxs, mismatches, losses_mismatches, mismatchescontainer = [],[],[],[],[],[]                                                      
+        truthlabels=np.asarray(self.y_true, dtype=int) 
+        classes_ids=[k for k in enumerate(self.data.classes)]
+        predclass=np.asarray(self.pred_class)
+        for i, pred in enumerate(predclass):
+            where_truth=np.nonzero((truthlabels[i]>0))[0]
+            mismatch=np.all(pred!=where_truth)
+            if mismatch: 
+                mismatches_idxs.append(i)
+                losses_mismatches.append((losses[i][pred],i))
+            infotup=(i, pred, where_truth, losses[i][pred], np.round(self.probs[i], decimals=3)[pred], mismatch)
+            infolist.append(infotup)
+        mismatches = self.data.valid_ds[mismatches_idxs]
+        ordlosses=sorted(losses_mismatches, key = lambda x: x[0], reverse=True)
+        for w in ordlosses: ordlosses_idxs.append(w[1])
+        mismatches_ordered_byloss=self.data.valid_ds[ordlosses_idxs]
+        print(mismatches)
```

Comment:
Is this for debugging purposes or do you want to print it?

Reply:
No, if you allow me to, I'd like to retain that print statement. Indeed, it prints the `header` of the subset of `valid_ds` but just with the mismatched items, and I find it to be quite informative.
Obviously I'll remove it should it conflict with the library general guidelines. Thanks!